tests/wallet-automation/wallet-setup.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, BrowserContext } from '@playwright/test';
+import * as fs from 'fs/promises';
+import * as path from 'path';
+
+
+const typhonId = 'KFDNIEFADAANBJODLDOHAEDPHAFOFFOH';
+const url = `https://clients2.google.com/service/update2/crx?response=redirect&os=win&arch=x64&os_arch=x86_64&nacl_arch=x86-64&prod=chromiumcrx&prodchannel=beta&prodversion=79.0.3945.53&lang=ru&acceptformat=crx3&x=id%3D${typhonId}%26installsource%3Dondemand%26uc`;
+const downloadPath = path.resolve(__dirname, 'extensions');
+
+
+test('downloadFile test', async ({ page }) => {
+    await fs.mkdir(downloadPath, { recursive: true });
+
+    const downloadPromise = new Promise(async (resolve) => {
+        page.once('download', async (download) => {
+            const filePath = path.join(downloadPath, download.suggestedFilename());
+            await download.saveAs(filePath);
+            console.log(`file has been downloaded to: ${filePath}`);
+            resolve(filePath);
+        });
+    });
+
+    try {
+        await page.goto(url, {
+            waitUntil: 'domcontentloaded',
+            timeout: 10000
+        });
+    } catch (error) {
+        console.log('navigation caused an exception, likely due to immediate download:', 'directDownload');
+    }
+
+    // wait for the download to complete
+    const downloadedFilePath = await downloadPromise;
+        
+    // verify the file exists
+    try {
+        await fs.access(downloadedFilePath as string); // Type assertion to string
+        console.log('file verification succeeded, file exists.');
+    } catch {
+        console.error('file verification failed, file does not exist.');
+        throw new Error('downloaded file does not exist.');
+    };
+
```

Comment:
You should put the code to unzip the crx here. You dont need to do an external python file, you can use the package `unzip-crx-3` Ex:
```
const unzip = require("unzip-crx-3");

    const crxFile = `${downloadedFilePath}`;

    unzip(crxFile).then(() => {
    console.log("Successfully unzipped your crx file..");
    });
```

Reply:
the unzip-crx-3 broke the repo for now maybe I did it wrong can you check? commit c94be62e80dd3b58d167f436f53f3b607af90dee